widgets/contact/contact.php

```diff
@@ -846,13 +852,18 @@ function contact_form_action( $instance, $storage_hash ){
 			}
 		}
 
+		// Add in the default sender if no email field is defined 
+		if ( ! isset( $email_fields['email'] ) && ! empty( $instance['settings']['default_sender'] ) ) {
+			$email_fields['email'] = $instance['settings']['default_sender'];
```

Comment:
hm... 
the behavior before has been, that the email sender has been just the webserver instance. 

I don't like the idea to put the users email here - as a mail from user@domain.com to user@domain.com sent from a webserver could typically create more spam points than www-data@domain.com to user@domain.com.

So somewhere there is already a catchall which puts a "valid" sender adress in the smtp traffic; 
I just wanted to have control to overwrite this if I wanted, so my patch would not change the default behaviour the contact form did have before.